show/main.py

```diff
@@ -172,8 +171,8 @@ def alias(interfacename):
 
     click.echo(tabulate(body, header))
 
-# 'summary' subcommand ("show interfaces summary")
-@interfaces.command()
+# 'summary' subcommand ("show interfaces summary") -- called if no subcommands are passed
```

Comment:
Comment is not correct - command no longer called `summary`

Reply:
Not sure i got this one -- 'summary' is indeed the sub-command name...

Reply:
You're correct; my mistake -- I think I was focused on the `default` command behavior.